account_mail_templates/i18n/lt.po

```diff
@@ -1,19 +1,14 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-#	* account_mail_templates
+# 	* account_mail_templates
 #
```

Comment:
What about translations to other languages? Not needed anymore?

Reply:
Well the only translated one was LV, but its outdated now, so its not clear what to put there. If needed, we can get translations later.